libafl_frida/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "libafl_frida"
 version = "0.2.0"
-authors = ["s1341 <github@shmarya.net>"]
+authors = ["s1341 <github@shmarya.net>", "OmreeBenari <github@Omreeb.net>"]
```

Comment:
This fields is for the author, not contributors. @domenukk may we should add a contributors section in the readme, like the one in aflpp.

Reply:
I don't think we need a specific contributors section, it's always more out of date and less accurate than the github contributors section, right?
WRT author, @s1341 and @OmreeBenari can figure out what seems right in this case.